crates/nu-cli/tests/commands/cd.rs

```diff
@@ -432,3 +432,25 @@ fn valuesystem_path_not_found() {
         assert!(actual.err.contains("No such path exists"));
     })
 }
+
+#[cfg(target_os = "windows")]
+#[test]
+fn test_change_windows_drive() {
+    Playground::setup("cd_test_20", |dirs, sandbox| {
+        sandbox.mkdir("test_folder");
+
+        let actual = nu!(
+            cwd: dirs.test(),
+            r#"
+                pwd | config --set_into main_dir 
```

Comment:
You should be able to get the `pwd` from outside the test, so you don't have to store it into the config.

@andrasio - do you know if there are handy shortcuts in the test system you could use to do something like this?

Reply:
@jonathandturner I saved it in the config so that I can go out of Z drive to run `subst Z: /d` to delete the virtual drive.  

I guess I could do something like 
```shell 
cd ~
subst Z: /d
```
to delete the Z drive? 

Then assert that test_folder\test_file.txt exists, instead of checking for its contents?

Reply:
That's even better than the ideas I was coming up with. Yes, that should work great.

Reply:
I've made the changes 😄